js/index.js

```diff
@@ -21,4 +21,19 @@ menuItems.forEach(
   (pageLink) => {
     pageLink.addEventListener('click', toggleMenu);
   },
-);
\ No newline at end of file
+);
+
+// form validation//
+const form = document.getElementById('form');
+const email = document.getElementById('email');
+const emailRegex = /^[a-z@._0-9]*$/;
+const invalidEmail = document.querySelector('.error-msg');
+
+form.addEventListener('submit', (e) => {
+  if (!email.value.match(emailRegex)) {
+    invalidEmail.textContent = 'Please type the email in lower case!';
+    e.preventDefault();
+    return false;
+  }
+  return true;
+});
```

Comment:
- [ ] Good job so far your validation form is working as expected :+1:  Please ensure that the **error message** should show up close to the **"submit" button.** because is part of the requirement. :smile:  in your case, your error message shows up close to the input. 

![Screenshot from 2022-07-22 10-09-18](https://user-images.githubusercontent.com/6615394/180418207-9aef1830-4599-4a83-82a7-0fc3ae7d1a9d.png)